### ISSUE TITLE
Logic possibly broken on master due to wrongfully placed unicode glyph?

### ISSUE THREAD

**Issue by exilef** [OpenVisualCloud/SVT-AV1#373]
Hi, a checkout of the master branch ca23754aa22149b6ebbca8baae01a5b3e525dcdc seems to have a unicode glyph error in it that potentially leads to a logical error.

Compile warning:
```
Source/Lib/Common/Codec/EbDefinitions.h:2463:13: warning:
      treating Unicode character <U+FF01> as identifier character rather than as
      '!' symbol [-Wunicode-homoglyph]
        if (！pointer) return return_type;
```

Instead of checking for a null pointer via the not operator (exclamation mark symbol) followed by a space, the Unicode symbol "！" somehow slipped into the source. You might want to fix this, as I am not sure whether this might result in a (potentially compiler-dependent) implicit reversal of the logical expression inside the `if` statement above..

Did not have time to check, but this looks as it might cause trouble.

**Comment by xuguangxin** [comment]
HI @exilef , which compiler you used, I tried Clang 6.0.0 and GCC 7.3.0.
I did not find the warning.

**Comment by exilef** [comment]
AppleClang 10.0.1.10010046

**Comment by jbeich** [comment]
Can you reproduce after #371?

**Comment by JackZhouVCD** [comment]
fixed by #371
